hbase-client/src/main/java/org/apache/hadoop/hbase/DroppedSnapshotException.java

```diff
@@ -40,4 +40,15 @@ public DroppedSnapshotException() {
   public DroppedSnapshotException(String message) {
     super(message);
   }
+
+  /**
+   * DroppedSnapshotException with cause
+   *
+   * @param message the message for this exception
+   * @param cause the cause for this exception
+   */
+  public DroppedSnapshotException(String message, Throwable cause) {
+    super(message, cause);
+  }
+
```

Comment:
NIT: Could you please remove the empty line (also in the other files).

Reply:
Done